helpers/test-helper.js

```diff
@@ -90,14 +90,14 @@ async function checkError(promise, errMsg, isAssert) {
   assert.equal(receipt.status, 0x00);
 
   if (isAssert) {
-    const network = await web3GetNetwork();
     const transaction = await web3GetTransaction(tx);
-    if (network !== "coverage") {
-      // When a transaction `throws`, all the gas sent is spent. So let's check that we spent all the gas that we sent.
-      // TODO: Properly account for EIP150 and passing only 63/64 gas on CALL.
-      // See https://github.com/JoinColony/colonyNetwork/issues/194 for details.
-      assert.closeTo(transaction.gas, receipt.gasUsed, 95000, "didnt fail - didn't throw and use all gas");
-    }
+    // As of EIP150, when a CALL is made only 63/64 on the remaining gas is passed on to to the called contract.
+    // If an assert fails, that gas is expected to be spent.
+
+    const remainingBeforeCall = (transaction.gas - receipt.gasUsed) * 64;
+    const etherRouterGas = transaction.gas - remainingBeforeCall;
```

Comment:
Is `etherRouterGas` this the same as `receipt.gasUsed`? And how is this `etherRouterGas`?